titannet/chain.json

```diff
@@ -0,0 +1,126 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "titannet",
+  "chain_type": "cosmos",
+  "chain_id": "titan-test-4",
```

Comment:
Is this a testnet?

Reply:
> Is this a testnet?

Yes, this is indeed a testnet (titan-test-4) as shown in the chain.json configuration.
We're in the final preparation phase before mainnet launch. At this stage, we aim to test IBC channel connectivity between our Testnet and OSMO, particularly focusing on TNT4 cross-chain transfers to OSMO to validate additional functionalities.
Both our team and community are fully aware of its testnet status. We kindly request approval for this submission. We're open to any modification requirements or guidelines you may have and can adjust according to your specifications. What are your requirements for this integration?

Reply:
In this case, the value for `"network_type"` shouldn't be `"mainnet"`, but instead should be `"testnet"`.
In addition, the chain_name should be titannettestnet (in the assetlist.json, the chain.json, versions.json, all ibc files, and the directory name. The directory must also be relocated inside the /testnets/ directory, where you'll see that all other testnets have chain_name as <mainnet name>+'testnet'.